sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormContentType.java

```diff
@@ -10,6 +10,12 @@
  */
 public final class FormContentType extends ExpandableStringEnum<FormContentType> {
 
+    /**
+     * Creates a FormContentType object.
+     */
+    public FormContentType() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.